commit 43ac5f61a348ef615582be86b541949f1199e252
Author: Dave Pacheco <dap@joyent.com>
Date:   2019-04-05T16:07:13-07:00 (6 months ago)
    
    joyent/mdb_v8#117 repo cleanup from recent change
    Reviewed by: Ashwin Nair <nairashwin952013@gmail.com>
    Approved by: Ashwin Nair <nairashwin952013@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index fb0bbe7..9a7bac3 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -5,12 +5,12 @@
 -->
 
 <!--
-    Copyright (c) 2018, Joyent, Inc.
+    Copyright (c) 2019, Joyent, Inc.
 -->
 
 # mdb_v8 changelog
 
-## v1.4.0 (2018-03)
+## v1.4.0 (2019-04)
 
 * #107 would like dcmd for heuristically finding back references
 * #111 want `::v8whatis`
diff --git a/README.md b/README.md
index a56ea2b..2866dd0 100644
--- a/README.md
+++ b/README.md
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2015, Joyent, Inc.
+    Copyright (c) 2019, Joyent, Inc.
 -->
 
 # mdb_v8: postmortem debugging for Node.js
@@ -44,17 +44,17 @@ https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8.  If you have the
 can list the latest binaries with:
 
     $ mfind -t o $(mget -q /Joyent_Dev/public/mdb_v8/latest)
-    /Joyent_Dev/public/mdb_v8/v1.3.0/mdb_v8_amd64.so
-    /Joyent_Dev/public/mdb_v8/v1.3.0/mdb_v8_ia32.so
+    /Joyent_Dev/public/mdb_v8/v1.4.0/mdb_v8_amd64.so
+    /Joyent_Dev/public/mdb_v8/v1.4.0/mdb_v8_ia32.so
 
 You can fetch a specific binary like this (in this case, the 32-bit version
-1.3.0 binary):
+1.4.0 binary):
 
-    $ mget -O /Joyent_Dev/public/mdb_v8/v1.3.0/mdb_v8_ia32.so
+    $ mget -O /Joyent_Dev/public/mdb_v8/v1.4.0/mdb_v8_ia32.so
 
 or using curl:
 
-    $ curl -O https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8/v1.3.0/mdb_v8_ia32.so
+    $ curl -O https://us-east.manta.joyent.com/Joyent_Dev/public/mdb_v8/v1.4.0/mdb_v8_ia32.so
 
 This one-liner will get you the latest 32-bit binary:
 
